DataFormats/Headers/include/Headers/DataHeader.h

```diff
@@ -546,12 +570,24 @@ struct DataHeader : public BaseHeader
   ///
   /// sub specification (e.g. link number)
   ///
-  SubSpecificationType subSpecification;
+  SubSpecificationType equipmentID;
 
   ///
-  /// size of the associated data
+  /// index of the split payload
   ///
-  uint64_t    payloadSize;
+  SplitPayloadIndexType splitPayloadIndex;
+
+  ///
+  /// size of the associated data buffer
+  ///
+  PayloadSizeType payloadSize;
+
+  ///
+  /// TimeframeID this data belongs to
+  ///
+
+  uint64_t timeframeStart;
+  uint64_t timeframeDuration;
```

Comment:
Timeframe duration is defined as multiple of HBF durations (orbits).
Worst case it's a run parameter. Or am I missing the point of this field?

Reply:
We could have quantities that span multiple runs, e.g. calibrations, QA histograms etc.